pkg/rancher-desktop/main/tray.ts

```diff
@@ -183,6 +190,14 @@ export class Tray {
     this.buildFromConfig();
     this.watchForChanges();
 
+    /**
```

Comment:
This should probably be a plain comment (`/*` / `//`) instead of a JSDoc comment (`/**`), because it's not at the declaration of the variable (nor should it be).

Reply:
I was just being consistent with the comment on line 206 for the `networkInterval`

Reply:
Nevermind, I'll put a commit together that replaces that one as well.

Reply:
Yeah, the existing code doesn't make sense either :)